charts/ratify/README.md

```diff
@@ -0,0 +1,114 @@
+# Ratify Helm Chart
+
+## Get Repo Info
+
+```console
+helm repo add ratify https://deislabs.github.io/ratify
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+```console
+# Helm install with gatekeeper-system namespace already created
+$ helm install [RELEASE_NAME] ratify/ratify --atomic --namespace gatekeeper-system
+
+# Helm install and create namespace
+$ helm install [RELEASE_NAME] ratify/ratify --atomic --namespace gatekeeper-system --create-namespace
+
+```
+
+_See [parameters](#parameters) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Upgrade Chart
+
+```console
+$ helm upgrade -n gatekeeper-system [RELEASE_NAME] ratify/ratify
+```
+
+## Parameters
+
+| Parameter                                          | Description                                                                                                                                                                                                                         | Default                        |
+| -------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------ |
+| image.repository                                   | Ratify app image                                                                                                                                                                                                                    | `ghcr.io/deislabs/ratify`      |
+| image.crdrepository                                | Ratify CRD install Image                                                                                                                                                                                                            | `ghcr.io/deislabs/ratify-crds` |
+| image.tag                                          | Image tag                                                                                                                                                                                                                           | `v1.0.0`                       |
```

Comment:
Maybe we need to remind updating the tag in the release doc?

Reply:
good point. I'm thinking if I should make the default value a generic message like [LATEST RELEASE TAG]. Since this increases the number of manual steps for release. We already have quite a few and it's easy to miss things like this. What do you think?

Reply:
yeah, I feel latest tag would help a lot since we have a few places pin to a specific version